syntax = "proto3";

package sample;

service HelloWorldService {
  rpc HelloWorld(HelloWorldRequest) returns (HelloWorldResponse);
}

message HelloWorldRequest {}

message HelloWorldResponse {
  string message = 1;
}
